compiler/rustc_trait_selection/src/traits/query/normalize.rs

```diff
@@ -292,7 +344,7 @@ impl<'cx, 'tcx> TypeFolder<'tcx> for QueryNormalizer<'cx, 'tcx> {
                 )
             }
 
-            _ => ty,
+            _ => ty.super_fold_with(self),
```

Comment:
Are you missing a `ty.super_fold_with(self)` on line 334 of this file?

Also, the `replace_placeholders` call on line 337, that's a no-op in the case of `Err`, right?

Reply:
Yes, I've missed a couple.

And yes, it's a no-op on Err. I'll move it.